counters.go

```diff
@@ -14,6 +14,11 @@ func NewCounter(timestamp Nanotime, value int64, source Source, tags Tags) Count
 	return Counter{Value: value, Timestamp: timestamp, Source: source, Tags: tags.Copy()}
 }
 
+func (c *Counter) AddTagsSetSource(additionalTags Tags, newSource Source) {
```

Comment:
Does this operation need to be threadsafe?

Reply:
Nope